docs/src/index.md

```diff
@@ -38,6 +38,17 @@ julia> @btime sin(x) setup=(x=rand())
 0.49587200950472454
 ```
 
+If you want to benchmark an expression and store the value in a variable, you can use:
+
+```julia
+julia> y = @btime sin(x) setup=(x=rand())
+  3.467 ns (0 allocations: 0 bytes)
+0.3812650449427549
+
+julia> y
+0.3812650449427549
+```
+
```

Comment:
`@btime` docstring already documents that it returns the value, as `@time` also does, but I guess adding an explicit example doesn't hurt (which could also be added to `@btime` docstring directly instead).  This specific example, however, doesn't look extremely useful to me: you're taking the sine of a random value, well, one of the many several random values being generated for the benchmark.  Is that likely to be a realistic situation?